ports/rp2/mpconfigport.h

```diff
@@ -146,6 +146,12 @@
 #define MICROPY_VFS                             (1)
 #define MICROPY_VFS_LFS2                        (1)
 #define MICROPY_VFS_FAT                         (1)
+#ifndef MICROPY_PY_USOCKET
+#define MICROPY_PY_USOCKET                      (0)
+#endif
+#ifndef MICROPY_PY_NETWORK
+#define MICROPY_PY_NETWORK                      (0)
+#endif
```

Comment:
since they default to disabled, these 6 lines can be omitted

Reply:
Wouldn't be better if it's explicitly set to 0 here ?

Reply:
Why?  They are already disabled, and individual boards can enable them if needed.  Instead of having lots of unnecessary lines in mpconfigport.h we are trying to only include things that change the defaults.  See also #7655.

Reply:
Okay removed.